vm-systemd/misc-post-stop.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
-# Save default applications for DispVM
+# cleanup QubesIncoming/
+if [ -d /home/user/QubesIncomming ]; then
+    find /home/user/QubesIncoming -mindepth 1 -type f -empty -delete
+    find /home/user/QubesIncoming -mindepth 1 -type d -empty -delete
```

Comment:
Save the path (with the above adjustments) in a variable to not repeat it each time?